lib/util.js

```diff
@@ -72,6 +71,78 @@ const errorToString = Error.prototype.toString;
 var CIRCULAR_ERROR_MESSAGE;
 var Debug;
 
+// eslint-disable-next-line
+const escapeSequencesRegExp = /[\\'\x00-\x1f]/g;
+const keyStrRegExp = /^[a-zA-Z_][a-zA-Z_0-9]*$/;
+const textRegExp = /^[a-zA-Z_0-9 ,.?!+\-&"öäüÜÖÄß():;<>{}[\]=$]*$/;
+const colorRegExp = /\u001b\[\d\d?m/g;
+const numberRegExp = /^(0|[1-9][0-9]*)$/;
+
+// Table of character substitutions for text
+const escapeCodesText = {
+  39: "\\'",
```

Comment:
I wonder if using a sufficiently wide array (at least 128 to cover all of ASCII) would improve performance compared to a plain object like this. If not, it might be good to also compare against an object created with `Object.create(null)`.